gradle/depend.gradle

```diff
@@ -201,6 +201,7 @@ ext {
     biometric = "androidx.biometric:biometric:$biometric_version"
     preference = "androidx.preference:preference:$preference_version"
     core_ktx = "androidx.core:core-ktx:$core_ktx_version"
+    splashscreen = "androidx.core:core-splashscreen:1.0.0"
```

Comment:
come mai ha la versione hardcoded?

Reply:
ma non ho visto l'esigenza di avere la def in questo caso e abbastanza lineare non comune ad altre cose

Reply:
si era giusto per avere un uniformità con le altre dichiarazione delle dipendenze, ma va bene, approvo

Reply:
ho messo comunque la version